x-pack/plugins/lens/public/pie_visualization/suggestions.ts

```diff
@@ -232,11 +232,12 @@ export function suggestions({
         ],
       },
       previewIcon: 'bullseye',
-      hide:
-        groups.length !== 2 ||
-        table.changeType === 'reduced' ||
-        hasIntervalScale(groups) ||
-        (state && state.shape === 'mosaic'),
+      hide: true,
+      /** The following code should be uncomment when Experimental flag will be removed from Mosaic chart **/
```

Comment:
Could you move these into a new issue "Get mosaic out of experimental" instead of having commented out code in here?

Reply:
np